lib/Basics/FileUtils.cpp

```diff
@@ -184,7 +184,7 @@ static void fillStringBuffer(int fd, std::string const& filename,
     if (result.reserve(chunkSize) != TRI_ERROR_NO_ERROR) {
       THROW_ARANGO_EXCEPTION(TRI_ERROR_OUT_OF_MEMORY);
     }
-    ssize_t n = TRI_READ(fd, result.end(), static_cast<TRI_read_t>(chunkSize));
+    TRI_read_t n = TRI_READ(fd, result.end(), static_cast<TRI_read_t>(chunkSize));
```

Comment:
I don't think this makes sense.
The signature for `read` on Linux is:
```
ssize_t read(int fd, void *buf, size_t count);
``` 
So the parameter type for the function return type and the call argument are different.
This PR makes things worse. What is the motivation for this?

Reply:
on linux `TRI_read_t` *is* `ssize_t`; However, ssize_t is not present on windows. 

as its name says, `TRI_read_t` is intended to be the type to be returned by `TRI_READ`.